README.md

```diff
@@ -318,7 +287,6 @@ Finally, thanks to the fusion and inlining of carriers, `fused-effects` is only
 
 [`mtl`]: http://hackage.haskell.org/package/mtl
 
-
 #### Comparison to `freer-simple`
```

Comment:
Double spaces preceding headings are intentional, as they make the document _much_ more readable for me. Please leave them in place.